desc/protoparse/lexer_test.go

```diff
@@ -237,6 +251,7 @@ func TestLexerErrors(t *testing.T) {
 		testutil.Eq(t, _ERROR, tok)
 		testutil.Require(t, sym.err != nil)
 		testutil.Require(t, strings.Contains(sym.err.Error(), tc.errMsg), "case %d: expected message to contain %q but does not: %q", i, tc.errMsg, sym.err.Error())
+		t.Logf("case %d: %v", i, sym.err)
```

Comment:
Did you intend to leave this?

Reply:
I did. Having `t.Logf` sprinkled in tests seems generally useful for future me, troubleshooting a test. And this isn't a ton of noise in the output, so it didn't seem egregious to leave (especially since it's test code... I would be way more diligent about deleting logs of questionable value if it were non-test code).